source/server/getting-started/launch-script.rst

```diff
@@ -30,6 +30,13 @@ Windows
 
 Save your Windows launch script as ``launch.bat``.
 
+.. warning::
+   Launching your server in a Command Prompt or PowerShell environment can lead to the server hanging. During this 
+   time, the only messages captured in the log state the server skipped many seconds or minutes worth of ticks. 
```

Comment:
This sentence is a little hard to parse. May I suggest changing it to read
...`messages captured in the log will state that the server skipped`...

Reply:
Changed

Reply:
It's reverted in the latest edit :/